README.md

```diff
@@ -57,6 +57,24 @@ The tooling has evolved over time and therefore when generating new packages, yo
 This is because the new package has a dependency on an existing package, it was regenerated and changes were detected from
 when it was originally generated.
 
+#### Backends
+The tooling support two backends for generating reference assemblies: [the Cci-based GenAPI](https://github.com/dotnet/arcade/tree/main/src/Microsoft.DotNet.GenAPI) and [the Roslyn-based GenAPI](https://github.com/dotnet/sdk/tree/main/src/GenAPI). The default one is Roslyn-based. You can switch between backends using the `--genapi-backend` parameter that supports `cci` and `roslyn` values.
+
+```bash
+./generate.sh --pkg system.buffers,4.5.1 --genapi-backend roslyn --dest <destination path>
+```
+
+The cci-based GenAPI has lack of support as Common Compiler Infrastructure (cci) is deprecated. On the other hand, the Roslyn-based GenAPI is built on top of `Microsoft.CodeAnalysis.Workspaces` and has better support for the latest language features. The cci-based GenAPI will be part of the SBRP generator as a fallback solution.
+
+#### Best practices
+Currently, the tooling cannot generate 100% compilable reference assemblies, and manual modification of the generated source code is required. The best approach is to:
```

Comment:
This duplicates / omits information provided in the `Reference` section of the `README.md`. I think it would benefit the reader if they were somewhat unified, especially if the best practices list were to be extended in the future